src/components/calltree/EmptyReasons.css

```diff
@@ -0,0 +1,10 @@
+.CallTreeEmptyReasons {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: var(--grey-20);
+  box-shadow: inset 0 0 15px rgba(0, 0, 0, 0.1);
+  padding: 10px;
+}
```

Comment:
I think I'd like to see the text slightly bigger.

Also on a big screen, being centered makes it quite far from where I'd be looking with some data: when there's a tree I start looking at the top of the tree, so here I feel like I'm scanning the page too much to find some information. But I don't have a good suggestion.

So I think we can land this so that then Victoria can try it "for real" and give suggestions about how to make it better.

Reply:
I made the font slightly bigger, which I think is a good call. Design-wise the text being in the top left feels a bit awkward to me, so I'll keep it as it is for now. Hopefully people will notice the color is a bit different on the background.